src/lib/models/game.ts

```diff
@@ -301,14 +313,12 @@ const movePiece = (
       .getMoves(piece)
       .find((m) => m.row === row && m.file === file);
```

Comment:
can add the exclamation(bang!) here in order to remove the need to repeat it later